packages/wonder-blocks-modal/components/standard-modal.js

```diff
@@ -58,28 +58,73 @@ type Props = {
  * The "standard" modal layout: a titlebar, a content area, and a footer.
  */
 export default class StandardModal extends React.Component<Props> {
+    /** A unique HTML ID for the title element. */
+    _titleId: string;
+
+    /** A unique HTML ID for the subtitle element. */
+    _subtitleId: string;
+
     static defaultProps = {
         onClickCloseButton: () => {},
     };
 
+    constructor(props: Props) {
+        super(props);
+
+        // Generate a unique ID for this modal. Math.random() isn't great, but
+        // collisions on the same page should be pretty unlikely.
+        const modalId = Math.random()
```

Comment:
Interesting how this randomness doesn't break snapshot tests. I suppose they must seed the PRNG!

Reply:
…oh, or maybe it's because we don't currently do snapshot testing for modals >_> Cross that bridge when we get there? :/